### ISSUE TITLE
Thanks!!!

### ISSUE THREAD

**Issue by Scope666** [Nibre/MotherVR#203]
Glad to see your back!  This has always been my favorite VR mod, and I still consider it the first AAA title that I've ever played.  Thanks for the Epic Store fix!!!    :-)

**Comment by g-coded** [comment]
Yes, glad to see you back!  I check on this and ReclaimerVR from time to time and was getting worried something happened.  I was shocked when I saw the '20 hours ago' in the releases.
